### PR TITLE
#10539 categorical.rst doc edit

### PR DESCRIPTION
**Pull request by Konntroll**
Finished editing categorical.rst for issue #10539. Might want to tweak a few things later on. In the meantime, please let me know if anything's off.

One thing to note is that a part of the 'Scatter' section seems to be missing. The section jumps straight to 'Adding jitter' although there is an example (categorical_scatter.py) referenced before that section. Looks like somebody 'commented' the example out and the source is missing from the 'examples' folder. If you can provide this example (and maybe some notes), I'll be happy to add a writeup for it.

### CODE REVIEW

**Comment by bryevdv** [comment]
I pushed a quick commit to fix the linter issues so this should be green now. @tcmetzger whenever you have a chance to look I think your review will be most valuable :)

File: sphinx/source/docs/user_guide/categorical.rst [hunk 1]
```diff
@@ -222,106 +205,81 @@ the year, change the ``vbar`` function call to use ``factor_cmap`` for the
...
-*dodge*.
-
-In this scenario, our data is not "tidy". Instead of a single table with
-rows indexed by factors ``(fruit, year)``, we have separate series for each
-year. We can plot all the year series using separate calls to ``vbar``, but
-since every bar in each group has the same ``fruit`` factor, the bars would
-overlap visually. We can prevent this overlap and distinguish the bars
-visually by using the :func:`~bokeh.transform.dodge` function to provide an
-offset for each different call to ``vbar``:
+Take a scenario with separate sequesnces of ``(fruit, year)`` pairs
```
Comment (tcmetzger) [inline sphinx/source/docs/user_guide/categorical.rst:219]: sequences

File: sphinx/source/docs/user_guide/categorical.rst [hunk 2]
```diff
@@ -222,106 +205,81 @@ the year, change the ``vbar`` function call to use ``factor_cmap`` for the
...
-~~~~~~~~~~~~~~~~~~~
-
-The above techniques for stacking and grouping may also be used together to
-create a stacked, grouped bar plot.
+Stacking and grouping
+~~~~~~~~~~~~~~~~~~~~~
 
-Continuing the example above with bars grouped by quarter, we might stack each
-individual bar by region.
+You can also combine the above technicques to create plots of stacked and
```
Comment (tcmetzger) [inline sphinx/source/docs/user_guide/categorical.rst:234]: techniques

File: sphinx/source/docs/user_guide/categorical.rst [hunk 3]
```diff
@@ -222,106 +205,81 @@ the year, change the ``vbar`` function call to use ``factor_cmap`` for the
...
-
 .. _userguide_categorical_bars_pandas:
 
 Pandas
 ~~~~~~
 
-`Pandas`_ is a powerful and common tool for doing data analysis on tabular and
-timeseries data in Python. Although it is not *required* by Bokeh, Bokeh tries
-to make life easier when you do.
+`Pandas`_ is a powerful and popular tool for analysing tabular and time
```
Comment (tcmetzger) [inline sphinx/source/docs/user_guide/categorical.rst:260]: analyzing (American spelling)

**Comment by tcmetzger** [comment]
Great work, thank you!
I left comments about some very small typos.

**Comment by Konntroll** [comment]
Thanks for spotting these, @tcmetzger. Fixed now.

**Review by bryevdv — APPROVED** [review]

**Pull request merged**